source/core/parameters_lagrangian.cc

```diff
@@ -1784,6 +1906,8 @@ namespace Parameters
     template class GridMotion<3>;
     template class InsertionInfo<2>;
     template class InsertionInfo<3>;
+    template class ParticleRayTracing<2>;
```

Comment:
Does the application really support 2d?

Reply:
I don't understand what you mean by that answer. My question is can you do 2D ray tracing of a 2D DEM simulation? The fact that y ou can do a pseudo 2D problem in 3D DEM is orthogonal to this in my opinion.

Reply:
Same here, but let's just make sure we have at least tried it once beforehand